docs/file-rules-kts.md

```diff
@@ -0,0 +1,20 @@
+# The `rules.kts` file
+
+The `rules.kts` file allows defining custom policy rules to automatically reviews scan findings and using a rules DSL based on Kotlin.
```

Comment:
review**~~s~~**
"...scan findings. Rules are written in a Kotlin-based DSL."

Reply:
done